nifi-frontend/src/main/frontend/apps/nifi/src/app/pages/flow-designer/ui/canvas/canvas.component.html

```diff
@@ -18,6 +18,17 @@
 <div class="flex flex-col h-full">
     <fd-header></fd-header>
     <div class="flex-1">
+        <mat-drawer-container #flowAnalysisDrawer class="flex-1" style="position: unset">
```

Comment:
I think we want to use a sidenav and not a drawer. Also, The background color is blending in with the expansion panels. I think we want to go with a darker color like:

```
    mat-sidenav {
        background-color: if(
                $is-dark,
                $supplemental-theme-surface-palette-darker,
                $supplemental-theme-surface-palette-lighter
        );
    }
```

Reply:
This change is included in my latest [patch](https://github.com/apache/nifi/pull/8974#discussion_r1710346976).